server/data/data-jmap/src/main/java/org/apache/james/jmap/api/access/AccessToken.java

```diff
@@ -48,6 +48,10 @@ public String serialize() {
         return token.toString();
     }
 
+    public UUID getToken() {
```

Comment:
A token can't really have a getToken method IMO. Or it should return `this`. Maybe it's more a `serialize` or `asUUID`